### ISSUE TITLE
False positive for `no-unused-array-method-return`

### ISSUE THREAD

**Issue by Sparticuz** [sindresorhus/eslint-plugin-unicorn#3431]
`no-unused-array-method-return` is flagging the use of `values()` on `drizzle-orm`, which I guess is probably technically correct, I should be checking it's return value, but it doesn't make sense for this specific rule. It's not an array method.

`no-unused-array-method-return`

```js
await database.insert(xProspects).values({ value1: "hello" }).where(eq(xProspects.id,1));
```
